src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -1653,7 +1653,7 @@ def test_subscription_level_what_if_ts(self, resource_group, resource_group_loca
 
 
 class DeploymentScriptsTest(ScenarioTest):
-    @ResourceGroupPreparer(name_prefix='cli_test_deployment_scripts')
+    @ResourceGroupPreparer(name_prefix='cli_test_deployment_scripts', location='brazilsouth')
```

Comment:
> location='brazilsouth'

May I ask why you added location in those test?

Reply:
Sure, there's a known issue in ARM that when a service endpoint has multiple locations (e.g. one endpoint that serves east us and west us), ARM returns duplicated resources during a List operation, in our case, brazil south has one endpoint and one location.

Reply:
Got it~ Thanks